### ISSUE TITLE
Cannot read property `name` of undefined

### ISSUE THREAD

**Issue by prabakaranfresh** [ember-template-lint/ember-template-lint#2912]
[`aria-query`](https://github.com/A11yance/aria-query/) released a new package [5.2.1](https://github.com/A11yance/aria-query/releases/tag/v5.2.1) which is throwing an error for `undefined` in elementsRoleMap get method. All the component invocations which are `undefined` elementRoles are throwing as errors.

https://github.com/A11yance/aria-query/releases/tag/v5.2.1

Since the package.json didn't have a lock, the following error is occurring in template files which has a component invocation

`-- error: Cannot read property 'name' of undefined undefined`

**Comment by boris-petrov** [comment]
I already opened an issue in [aria-query](https://github.com/A11yance/aria-query/issues/507).

**Comment by villander** [comment]
I got a fix with `pnpm resolutions`: 

```json
"pnpm": {
    "overrides": {
      "aria-query": "5.1.3"
    },
}
```

**Comment by kategengler** [comment]
Link to the offending line https://github.com/ember-template-lint/ember-template-lint/blob/aec896233d803be9612953605f0949102a750cb3/lib/rules/no-unsupported-role-attributes.js#L26

This is breaking ember-cli's CI as well as the super-rentals-tutorial build.

**Comment by bmish** [comment]
Looks like it should be an easy fix. Can we get a failing code sample?

**Comment by kategengler** [comment]
Conveniently, the tests for the rule are failing with floating dependencies in this repo's own CI https://github.com/ember-template-lint/ember-template-lint/actions/runs/5262654063/jobs/9512048400

**Comment by bmish** [comment]
Let's see if this fixes it: https://github.com/ember-template-lint/ember-template-lint/pull/2914

**Comment by ef4** [comment]
Ugh, aria-query has now released a 5.3 that appears to undo these changes in 5.2.1. Their changelog literally says not to try using 5.2.

This breaks us again.

**Comment by ef4** [comment]
Nevermind, false alarm! I accidentally was looking at a version of ember-template-lint before this PR.
